wireapp/wire-ios-mono
# feat: [MLS Migration] add conversation fields [WPB-5267]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-5267" title="WPB-5267" target="_blank"><img alt="Sub-task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10816?size=medium" />WPB-5267</a>  [iOS] Support `ciphersuite` and `epoch_timestamp` for conversations 
  </td></table>
  <br />
 
















----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

See ticket https://wearezeta.atlassian.net/browse/WPB-5267

### Solutions



### Dependencies (Optional)

Merges into feature branch.

Needs releases with:
- [x] PR not open yet ~~GitHub link to other pull request~~

### Testing

#### Test Coverage (Optional)


#### How to Test

- Use `diya` environmnet
- Team must support MLS
- Login
- Create a new group -> should have the new fields in conversation

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
